include/tvm/target/target.h

```diff
@@ -31,6 +31,7 @@
 #include <tvm/target/target_kind.h>
 
 #include <string>
+#include <unordered_map>
```

Comment:
Should probably remove this as it's not used in this file.

Reply:
We will send a follow up PR, that does this just in sake of forward progress. Thanks!